package.json

```diff
@@ -2,10 +2,15 @@
   "name": "react-flexview",
   "version": "1.0.1",
   "description": "A react component to abstract over flexbox",
-  "main": "index.js",
+  "files": [
+    "lib",
+    "react-flexview.d.ts"
+  ],
+  "main": "lib/FlexView.js",
+  "typings": "react-flexview.d.ts",
   "scripts": {
     "test": "mocha",
-    "build": "rm -rf lib && mkdir lib && node-sass src --importer sass-importer.js --include-path node_modules -o lib && babel src -d lib",
+    "build": "rm -rf lib && mkdir lib && node-sass src --importer sass-importer.js --include-path node_modules -o lib && babel src -d lib && flow-copy-source -v src lib",
```

Comment:
since `flow gen-flow-files` is experimental/broken, the easiest way to ship typings along with the package on npm is to copy the full source, including flow type labels

the order of typing resolution for flow goes like:
- types in the package itself
- typings from flow-typed
- fallback to `any`